src/AwesomeCMSCore/Modules/AwesomeCMSCore.Modules.Account/Repositories/AccountRepository.cs

```diff
@@ -31,29 +31,17 @@ public class AccountRepository : IAccountRepository
         public async Task<IEnumerable<UserViewModel>> UserList()
```

Comment:
This method is use in /Account/Index page. Does this change break my table ?

Reply:
NO.